terraform/test/terragrunt.hcl

```diff
@@ -1,11 +1,16 @@
 terraform {
-  source = "git::https://github.com/bcgov/startup-sample-project-aws-containers-terraform-modules.git//?ref=v0.0.4"
```

Comment:
I made the tag v0.2 so this can be updated

Reply:
Done